.vscode/launch.json

```diff
@@ -14,7 +14,7 @@
         "--extensionDevelopmentPath=${workspaceFolder}"
       ],
       "outFiles": [
-        "${workspaceFolder}/out/**/*.js"
+        "${workspaceFolder}/dist/**/*.js"
```

Comment:
This change allows the debugger to bind breakpoints we put in the typescript files.

Reply:
Sweet! Nice add.